File: programers_Lv1/2016년.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12901?language=javascript
function solution(a, b) {
  const weekday = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
  // 자바스크립트 요일 확인
  return weekday[new Date(`2016-${a}-${b}`).getDay()];
}

console.log(solution(5, 24));

File: programers_Lv1/2016년.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12901?language=python3
import datetime

def solution(a, b):
    weekday = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"]
    answer = weekday[datetime.datetime(2016, a, b).weekday()]
    return answer

print(solution(5, 24))
